src/Providers/LaravelServiceProvider.php

```diff
@@ -12,6 +12,12 @@
 
 namespace PHPOpenSourceSaver\JWTAuth\Providers;
 
+use Laravel\Octane\Events\RequestReceived;
+use Laravel\Octane\Events\TaskReceived;
+use Laravel\Octane\Events\TickReceived;
```

Comment:
The octane package is not installed, therefore these events could not be found by phpstan. Would it not be cleaner to add it as a dependency?

Reply:
I had a look at laravel-ignition, they don't have listed the mentioned events in ignoreErrors. 
In `phpstan.neon.dist` they have enabled `checkOctaneCompatibility`. Is this the reason why they don't have to list the errors or how did they archive this?

Reply:
That is a rule added in larastan see link below. It only checks for bindings where it is being passed the container instance directly.

https://github.com/nunomaduro/larastan/blob/c36c27e02ebd152b556b8b691e0c88cbeae122a1/docs/rules.md#octanecompatibilityrule

`laravel-ignition` added their `IgnitionServiceProvider` to the `excludePaths` in https://github.com/spatie/laravel-ignition/blob/main/phpstan.neon.dist which is how they managed to avoid adding the errors to the baseline. I think the way I have done it is cleaner and means that the `LaravelServiceProvider` is covered by phpstan still. It's up to you guys though.

Reply:
I have not seen the `IgnitionServiceProvider` in their `excludePaths`. Thanks for clarifying this :) 

I totally agree with you, your way is cleaner to exclude specific errors instead of the whole file.